Pot noodl patch 1 (#48)
UnofficialSamHub/MeadowLand: Pages/GUIDELINES.md

---
title: Submission Guidelines
has_children: false
nav_order: 3
---

{% include analytics.html %}

# The MeadowLand Guidelines ![birb](../assets/images/Birb.png)
## Resource Pack Guidelines
1. Do not submit packs without transparent backgrounds
2. Do not submit packs that are the same/identical to another pack
3. Do not submit low quality packs. This includes, but is not limited to:
- Packs with random images that do not fit into the Meadow format
- Packs without any effort put into them

> 4. Please also check that the pack works before submitting it, due to the way the meadow was made you are limited with the sprite size that you can use

Reviewer: ```suggestion
4. Please check that the pack works before submitting it, due to the way the Meadow was made you may not resize the spritesheet
```
